manifests/osfamily/debian.pp

```diff
@@ -7,6 +7,14 @@
     key        => {
       'id'     => '47B320EB4C7C375AA9DAE1A01054B7A24BD6EC30',
       'server' => 'pgp.mit.edu',
-    }
+    },
+    notify     => Notify['pc1_repo_force'],
```

Comment:
In at least one instance, the `Exec['apt_update']` was the last resource evaluated, after trying to install packages from PC1. `notify => Exec['apt_update']` wasn't enough to pull the priority forward.